shim/src/stdlib.js

```diff
@@ -1,55 +1,77 @@
-import { getIntrinsics } from "./intrinsics";
+import { getIntrinsics } from './intrinsics';
 
 export function getStdLib(sandbox) {
-    const intrinsics = getIntrinsics(sandbox);
-    return {
-        Array: { value: intrinsics.Array },
-        ArrayBuffer: { value: intrinsics.ArrayBuffer },
-        Boolean: { value: intrinsics.Boolean },
-        DataView: { value: intrinsics.DataView },
-        Date: { value: intrinsics.Date },
-        decodeURI: { value: intrinsics.decodeURI },
-        decodeURIComponent: { value: intrinsics.decodeURIComponent },
-        encodeURI: { value: intrinsics.encodeURI },
-        encodeURIComponent: { value: intrinsics.encodeURIComponent },
-        Error: { value: intrinsics.Error },
-        eval: { value: intrinsics.eval },
-        EvalError: { value: intrinsics.EvalError },
-        Float32Array: { value: intrinsics.Float32Array },
-        Float64Array: { value: intrinsics.Float64Array },
-        Function: { value: intrinsics.Function },
-        Int8Array: { value: intrinsics.Int8Array },
-        Int16Array: { value: intrinsics.Int16Array },
-        Int32Array: { value: intrinsics.Int32Array },
-        isFinite: { value: intrinsics.isFinite },
-        isNaN: { value: intrinsics.isNaN },
-        JSON: { value: intrinsics.JSON },
-        Map: { value: intrinsics.Map },
-        Math: { value: intrinsics.Math },
-        Number: { value: intrinsics.Number },
-        Object: { value: intrinsics.Object },
-        parseFloat: { value: intrinsics.parseFloat },
-        parseInt: { value: intrinsics.parseInt },
-        Promise: { value: intrinsics.Promise },
-        Proxy: { value: intrinsics.Proxy },
-        RangeError: { value: intrinsics.RangeError },
-        ReferenceError: { value: intrinsics.ReferenceError },
-        Reflect: { value: intrinsics.Reflect },
-        RegExp: { value: intrinsics.RegExp },
-        Set: { value: intrinsics.Set },
-        String: { value: intrinsics.String },
-        Symbol: { value: intrinsics.Symbol },
-        SyntaxError: { value: intrinsics.SyntaxError },
-        TypeError: { value: intrinsics.TypeError },
-        Uint8Array: { value: intrinsics.Uint8Array },
-        Uint8ClampedArray: { value: intrinsics.Uint8ClampedArray },
-        Uint16Array: { value: intrinsics.Uint16Array },
-        Uint32Array: { value: intrinsics.Uint32Array },
-        URIError: { value: intrinsics.URIError },
-        WeakMap: { value: intrinsics.WeakMap },
-        WeakSet: { value: intrinsics.WeakSet },
-
-        // TODO: Annex B
-        // TODO: other special cases
-    };
+
+  const intrinsics = getIntrinsics(sandbox);
+
+  return {
```

Comment:
wrong alignment causing the diff to be bigger that it is. The entire project is using 4 spaces, but this file is using 2.